joeflow/runner/dramatiq.py

```diff
@@ -19,7 +19,7 @@ def task_runner(*, task_pk, workflow_pk, countdown=None, eta=None, retries=0):
     )
 
 
-class RetryError(Exception):
+class RetryError(dramatiq.errors.Retry):
```

Comment:
Hm… I just read up the documentation for the build in exception. I think we can simply drop the custom implementation there and just use drmatiq's `Retry` instead.